.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           python -VV
           python -m pip install --upgrade pip
-          make dev-install
+          pip install -r requirements_dev.txt
```

Comment:
You removed the use of dev-install but left it in Makefile and it's still referenced in the README and CONTRIBUTING files. Was that all intentional?

Reply:
I'm hoping in a week or so `make dev-install` will work again when pip-tools is fixed.

Reply:
> I'm hoping in a week or so `make dev-install` will work again when pip-tools is fixed.


I don't think that's necessary. `make dev-install` is a target is for our convenience, and wasn't doing anything special in our CI builds. (The reason it calls `pip-sync` is to synchronize a virtualenv with the requirements files generated by `pip-compile`, but there's no venv to synchronize in the CI runner).